cltk/tokenize/sentence.py

```diff
@@ -28,18 +32,24 @@ class TokenizeSentence():  # pylint: disable=R0903
     ``TokenizeSentence('greek')``.
     """
 
+    missing_models_message = "TokenizeSentence requires the models to be installed in cltk_data. Please load the correct models."
+
     def __init__(self: object, language: str):
         """Lower incoming language name and assemble variables.
         :type language: str
         :param language : Language for sentence tokenization.
         """
         self.language = language.lower()
 
-        if self.language not in INDIAN_LANGUAGES :
+        # Workaround for Latin—use old API syntax to load new sent tokenizer
+        if self.language == 'latin':
+            self.lang_vars = LatinLanguageVars()
+            BasePunktSentenceTokenizer.__init__(self, language='latin', lang_vars=self.lang_vars)
```

Comment:
PY3 super() please

Reply:
super() cannot be used here because there is no TokenizeSentence is not a subclass. This part is tbh convoluted because I am trying to retain functionality for TokenizeSentence for Latin in the short term while other languages are still using it. Going to rethink that strategy.